src/main/ipfs/ipfs.ts

```diff
@@ -0,0 +1,133 @@
+import { Request } from "./request_json"
+import { Response } from "./response_json";
+import * as net from 'net'
+import { DaemonRequest } from "./classes";
+
+class IPFS {
+    HOST: string = 'localhost';
+    PORT: number = 7777;
+    
+    client: net.Socket;
+
+    constructor() {
```

Comment:
Should we take port number as an argument in the constructor?

Reply:
Done